Doc/library/platform.rst

```diff
@@ -53,7 +53,7 @@ Cross Platform
 
 .. function:: machine()
 
-   Returns the machine type, e.g. ``'i386'``. An empty string is returned if the
+   Returns the machine type, e.g. ``'AMD64'``. An empty string is returned if the
```

Comment:
Is it what returned on the AMD processors? On my computer it is 'x86_64'.

Update also the docstring of platform.machine().

Reply:
The original 'proper' name of what is also called the "x86-64" architecture is AMD64, as it was created by AMD and later adopted by Intel when Intel's IA-64 (Itanium) architecture failed in the market. What is returned depends (AFAIK) on the OS, not the CPU; Windows and many (most?) Linux distros call it `AMD64` internally, while Apple and some others call it `x86-64`. Running a freshly built from `main` Python, as well as 3.9 and 3.10 release builds, `platform.machine()` returns `AMD64` on my Windows system with a stock Intel i7-3730.

> Update also the docstring of platform.machine().

I can, but as this PR currently only modifies the docs, I'd rather do that separately; there are other places in the codebase that should be updated too, for consistency.

Reply:
Docstrings are a part of the docs. If we do not update them together with the rst files, they are left desynchronized.

Reply:
This only affects the choice of one specific example, both of which are equally accurate and valid, and which will be synchronized if and when I do a similar pass through the codebase itself. Several of the other `platform` functions, e.g. `system()`, have differing examples on each. And given the rest of this PR scrupulously avoids touching the code, adding this one trivial change has non-trivial cost, of triggering and requiring a whole suite of extra builds/CIs, and increasing risk for backporting.

Given the change was minor and not strictly required in the first place, and I almost didn't make it, if this is going to be an issue it'll make much more sense for me to just drop the change instead, since its not at all worth the cost.

Reply:
Either way the docstrings and the documentation should be consistent.